MapboxDirections/MBRouteOptions.swift

```diff
@@ -442,12 +442,12 @@ open class RouteOptions: NSObject, NSSecureCoding, NSCopying{
         
         if !roadClassesToAvoid.isEmpty {
             let allRoadClasses = roadClassesToAvoid.description.components(separatedBy: ",")
-            let firstRoadClass = String(describing: allRoadClasses.first)
             if allRoadClasses.count > 1 {
                 assert(false, "`roadClassesToAvoid` only accepts one `RoadClasses`.")
             }
-            params.append(URLQueryItem(name: "exclude", value: firstRoadClass))
-
+            if let firstRoadClass = allRoadClasses.first {
+                params.append(URLQueryItem(name: "exclude", value: String(describing: firstRoadClass)))
```

Comment:
Is it necessary to essentially convert this string into a string?

Reply:
Guess not.